homeworks/vladislav.buzunar_buzyaka/4-objects/objects.js

```diff
@@ -0,0 +1,44 @@
+const tamagochi = {
+    name: 'Tamagochi',
+    age: 23,
+    healthLevels: 5,
+    happyLevels: 3,
+    foodLevels: 4,
```

Comment:
Please, create one additional field called alive, indicating if the Tamagotchi is alive.
You can do it by either changing the field value into the check method or using a getter.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get

If the Tamagotchi is dead, any method should not be called and logs that Tamagotchi is dead and you cannot play/eat/do excersices